bitcoin/src/taproot.rs

```diff
@@ -1318,9 +1318,21 @@ impl FutureLeafVersion {
         }
     }
 
-    /// Returns the consensus representation of this [`FutureLeafVersion`].
+    /// Returns the consensus encoded `u8` representation of this [`FutureLeafVersion`].
     #[inline]
-    pub fn to_consensus(self) -> u8 { self.0 }
+    pub fn to_consensus_u8(self) -> u8 { self.0 }
+
+    /// Returns the consensus representation of this [`FutureLeafVersion`] as a `u32`.
+    #[inline]
+    pub fn to_consensus_u32(self) -> u32 { self.to_consensus_u8().into() }
```

Comment:
I don't think this is used anywhere? We should remove `_32` methods for `FutureLeafVersions`